apis/openstack-nova/src/main/java/org/jclouds/openstack/nova/v2_0/config/NovaHttpApiModule.java

```diff
@@ -96,6 +94,8 @@ private void bindDefaultAliases() {
             URI.create("http://docs.openstack.org/compute/ext/os-volume-attachment-update/api/v2"));
       aliases.addBinding(URI.create(ExtensionNamespaces.ATTACH_INTERFACES)).toInstance(
             URI.create("http://docs.openstack.org/compute/ext/interfaces/api/v1.1"));
+      aliases.addBinding(URI.create(ExtensionNamespaces.HYPERVISORS)).toInstance(
+            URI.create("http://docs.openstack.org/compute/ext/hypervisors/api/v1.1"));
```

Comment:
no idea why we cant use ExtensionNamespaces.HYPERVISORS instead of specifying the URI again ...but since all the other binding do the same, i am following the same pattern FYI